test/accreditation/generate_data.py

```diff
@@ -98,23 +98,21 @@ def generate_data_noisy():
     accsys = make_accred_system(seed=seed_accreditation)
     simulator = qiskit.Aer.get_backend('qasm_simulator')
     noise_model = NoiseModel()
-    p1q = 0.002
+    p1q = 0.0002
     noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), 'u1')
     noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), 'u2')
     noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), 'u3')
-    p2q = 0.02
+    p2q = 0.005
     noise_model.add_all_qubit_quantum_error(depolarizing_error(p2q, 2), 'cx')
 
     basis_gates = ['u1', 'u2', 'u3', 'cx']
```

Comment:
Note that these basis gates are deprecated. Consider using instead:
     `basis_gates = ['cz', 'sx', 'x', 'cx']`